assets/aml-benchmark/components/inference-postprocessor/spec.yaml

```diff
@@ -0,0 +1,94 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+type: command
+
+name: inference_postprocessor
+display_name: Inference Postprocessor
+description: Inference Postprocessor
+version: 0.0.19
```

Comment:
I guess we can start with version `0.0.1` instead of `0.0.19`.

Reply:
Got it.